.github/workflows/release-sdk.yml

```diff
@@ -43,3 +38,22 @@ jobs:
         uses: svenstaro/upload-release-action@v2
         with:
           file: RadarSDK.xcframework.zip
+      
+      # get sha256 checksum of the XCFramework
+      - name: Get SHA256 checksum
+        id: checksum
+        run: echo "::set-output name=checksum::$(shasum -a 256 RadarSDK.xcframework.zip | cut -d ' ' -f 1)"
+
+      - name: Repository Dispatch
+        if: ${{ !github.event.release.prerelease }}
+        uses: peter-evans/repository-dispatch@v3
+        with:
+          token: ${{ secrets.SPM_REPO_KEY }}
+          repository: radarlabs/radar-sdk-ios-spm
+          event-type: update-xcframework
+          client-payload: '{"release": "${{ github.event.release.tag_name }}", "checksum": "${{ steps.checksum.outputs.checksum }}"}'
```

Comment:
Can we auto-create the release on `radar-sdk-ios-spm`? Ideally we don't create a manual step of having to merge a PR in that repo with this.

Reply:
This is mainly based off example snippets from: https://github.com/peter-evans/repository-dispatch

We do something similar in https://github.com/radarlabs/radar-sdk-ios/blob/master/.github/workflows/downstream-version-bump.yml, but if you look at the our github action history that action has been failing for a year now. I personally don't think that automation is useful but that is another topic.

Reply:
And yes, we can automate merging and creating releases.